### PR TITLE
Change doc example to use set_* methods.

### PR DESCRIPTION
**Pull request by ian-hamlin**
Hi,

I just noticed the documentation was now slightly wrong with the new version, this change fixes that.

### CODE REVIEW

**Comment by ian-hamlin** [comment]
> Good catch! The test is marked `ignore` to avoid actually saving a file when running tests, maybe it should be marked `no-run` instead, to make sure it stays compilable after the fix?

That's an excellent idea, I've made the change.

**Comment by ian-hamlin** [comment]
That broke one part of the build, so I went back to ignore for now :)

File: src/lib.rs [hunk 1]
```diff
@@ -25,15 +25,14 @@
 //! use std::path::Path;
```
Comment (HeroicKatora) [inline src/lib.rs]: It is possible to wrap this whole test in a hidden `#[cfg(…)]` even in a doctest (note the preceding `#`). That should exclude the block when the `encoder` feature is not enabled and allow you to use `no_run` again.

```
//! # #[cfg(feature = "encoder")] {
…
//! # }
```

**Comment by HeroicKatora** [comment]
Just noticed that I approved these but forgot to merge. Thank you!

**Pull request merged**